src/main/java/com/puppycrawl/tools/checkstyle/gui/ParseTreeTablePModel.java

```diff
@@ -182,7 +182,14 @@ public int getChildCount(Object parent) {
             result = ((DetailNode) parent).getChildren().length;
         }
         else {
-            result = ((DetailAST) parent).getChildCount();
+            if (parseMode == ParseMode.JAVA_WITH_JAVADOC_AND_COMMENTS
+                    && ((AST) parent).getType() == TokenTypes.COMMENT_CONTENT
+                    && JavadocUtils.isJavadocComment(((DetailAST) parent).getParent())) {
+                result = 1;
```

Comment:
please explain this "1" in single line comment, there are so much certain conditions that point that there will be new tree but result is "1".